.github/workflows/sync-docs.yaml

```diff
@@ -0,0 +1,72 @@
+name: Sync docs to docs site repo
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+  workflow_dispatch:
+
+jobs:
+  config-sync:
+    name: Synchronize docs to docs site repo
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.merged }}
```

Comment:
```suggestion
```

I haven't tried this, but I'm wondering how this works with `workflow_dispatch`? 🤔